spec/graphql/schema/printer_spec.rb

```diff
@@ -63,14 +63,47 @@
       end
     end
 
-    GraphQL::Schema.define(query: query_root, resolve_type: :pass)
+    create_post_mutation = GraphQL::Relay::Mutation.define do
+      name "CreatePost"
+      description "Create a blog post"
+
+      input_field :title, !types.String
+      input_field :body, !types.String
+
+      return_field :post, post_type
+
+      resolve -> (_, _, _) { }
```

Comment:
[There is RuboCop already](https://github.com/rmosolgo/graphql-ruby/blob/master/.rubocop.yml) but [that rule doesn't exist yet](https://github.com/bbatsov/rubocop/issues/3631).